docs/usage-guide/topics/ch05-connection.md

```diff
@@ -1,5 +1,7 @@
 # TLS Connections
 
+Users must configure a connection, and associate it with a config. `s2n_connection` is responsible for managing the actual state of a TLS connection. In a TLS server, there will be one `s2n_connection` for each TCP stream. For each `s2n_config`, there may be many `s2n_connection` structs associated with it.
```

Comment:
Is this change needed? It seems like most of this is already stated in this section or the config section.